easybuild/easyblocks/g/gcc.py

```diff
@@ -1043,4 +1048,42 @@ def make_module_req_guess(self):
             'LD_LIBRARY_PATH': ['lib', 'lib64'],
             'MANPATH': ['man', 'share/man']
         })
+
+        # Install rpath wrappers, if built with RPATH enabled
+        if build_option('rpath'):
+            def create_rpath_wrappers(targetdir):
+                set_up_configuration(silent=True)
+                tc = Toolchain(name="system", version="0.1")
+
+                # Expect rpath_filter_dirs and rpath_include_dirs as environment variables of
+                # ,-separated lists by default. We might want to allow different separators.
+                # E.g. ":" could be used to get a similar syntax to PATH and LD_LIBRARY_PATH
+                separator = ','
+                rpath_filter_dirs = None
+                if "RPATH_FILTER_DIRS" in os.environ:
+                    rpath_filter_dirs = os.environ["RPATH_FILTER_DIRS"].split(separator)
+                rpath_include_dirs = None
+                if "RPATH_INCLUDE_DIRS" in os.environ:
+                    rpath_include_dirs = os.environ["RPATH_INCLUDE_DIRS"].split(separator)
```

Comment:
Don't we want these environment variables to be picked up when the wrapper script is being used?

As it is now, you're picking up `$RPATH_FILTER_DIRS` when `GCCcore` is being installed, and baking that into the wrapper.

Allowing for this will require some work in the script itself (in `easybuild-framework`) though...

So for now, I would just call `prepare_rpath_wrappers` without passing anything?

Reply:
I've removed this for now, since this should go into the wrapper script PR in easybuild-framework